rosdep/base.yaml

```diff
@@ -2481,6 +2485,8 @@ libqt5-widgets:
   slackware: [qt5]
   ubuntu: [libqt5widgets5]
 libqt5x11extras5-dev:
+  debian: [libqt5x11extras5-dev]
+  fedora: [libQt5X11Extras-devel]
```

Comment:
It seems like i used the wrong package name. Now I found qt5-qtx11extras-devel (https://apps.fedoraproject.org/packages/qt5-qtx11extras-devel).